### ISSUE TITLE
Logstash does not start when using `11.0.15+10`, `17.0.3+7` versions of the JDK on Windows

### ISSUE THREAD

**Issue by robbavey** [elastic/logstash#14042]
**Logstash information**:

Logstash version (e.g. `bin/logstash --version`)
`logstash 8.1.3`
Also fails on `7.17.3`
Logstash installation source (e.g. built from source, with a package manager: DEB/RPM, expanded from tar or zip archive, docker)
Downloaded from elastic.co
How is Logstash being run (e.g. as a service/service manager: systemd, upstart, etc. Via command line, docker/kubernetes)
Via command line

**Plugins installed**: (`bin/logstash-plugin list --verbose`)
Stock

**JVM** (e.g. `java -version`):

If the affected version of Logstash is 7.9 (or earlier), or if it is NOT using the bundled JDK or using the 'no-jdk' version in 7.10 (or higher), please provide the following information:

JVM version (`java -version`)
```
openjdk version "11.0.15" 2022-04-19 LTS
OpenJDK Runtime Environment Zulu11.56+19-CA (build 11.0.15+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.56+19-CA (build 11.0.15+10-LTS, mixed mode)
```

JVM installation source (e.g. from the Operating System's package manager, from source, etc).
.msi downloaded from https://www.azul.com/downloads/?version=java-11-lts&os=windows&package=jdk

Value of the `LS_JAVA_HOME` environment variable if set.
`C:\Zulu\zulu-11`

**OS version** (`uname -a` if on a Unix-like system):
WIndows Server 2016 Datacenter

**Description of the problem including expected versus actual behavior**:

When trying to start logstash with a minimal (or even no) pipeline, Logstash will fail to start, due to a Permission Denied error

**Steps to reproduce**:

Starting logstash with a minimal or no pipeline will reproduce the error, eg: 

`C:\logstash-8.1.3>bin\logstash -e 'input { stdin {}}'`

**Provide logs (if relevant)**:

```
[FATAL] 2022-04-27 15:13:16.650 [main] Logstash - Logstash stopped processing because of an error: (EACCES) Permission denied - NUL
org.jruby.exceptions.SystemCallError: (EACCES) Permission denied - NUL
        at org.jruby.RubyIO.sysopen(org/jruby/RubyIO.java:1237) ~[jruby.jar:?]
        at org.jruby.RubyFile.initialize(org/jruby/RubyFile.java:365) ~[jruby.jar:?]
        at org.jruby.RubyIO.open(org/jruby/RubyIO.java:1156) ~[jruby.jar:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.jruby.lib.ruby.stdlib.rubygems.user_interaction.initialize(C:/logstash-8.1.3/vendor/jruby/lib/ruby/stdlib/rubygems/user_interaction.rb:645) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.ui.rg_proxy.initialize(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/ui/rg_proxy.rb:11) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.ui=(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler.rb:91) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.ui(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler.rb:87) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.rubygems_integration.validate(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/rubygems_integration.rb:68) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.source.path.validate_spec(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/source/path.rb:168) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.source.path.load_spec_files(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/source/path.rb:182) ~[?:?]
        at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1821) ~[jruby.jar:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.source.path.load_spec_files(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/source/path.rb:176) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.source.path.local_specs(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/source/path.rb:107) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.source.path.specs(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/source/path.rb:115) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.specs_for_source_changed?(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:559) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.specs_changed?(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:544) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.converge_paths(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:588) ~[?:?]
        at org.jruby.RubyArray.any?(org/jruby/RubyArray.java:4553) ~[jruby.jar:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.converge_paths(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:587) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.initialize(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:133) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.dsl.to_definition(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/dsl.rb:221) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.dsl.evaluate(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/dsl.rb:13) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition.build(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler/definition.rb:38) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.definition(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler.rb:197) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.vendor.bundle.jruby.$2_dot_5_dot_0.gems.bundler_minus_2_dot_3_dot_11.lib.bundler.setup(C:/logstash-8.1.3/vendor/bundle/jruby/2.5.0/gems/bundler-2.3.11/lib/bundler.rb:145) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.lib.bootstrap.bundler.setup!(C:/logstash-8.1.3/lib/bootstrap/bundler.rb:87) ~[?:?]
        at C_3a_.logstash_minus_8_dot_1_dot_3.lib.bootstrap.environment.<main>(C:\logstash-8.1.3\lib\bootstrap\environment.rb:89) ~[?:?]
```

Encountered this issue while attempting to run JDK tests for `jruby-9.3.x` upgrade. However, this issue is also present on released versions of Logstash.

Appears to be related to [this](https://github.com/jruby/jruby/issues/7182) issue in jruby
This appears to be related to https://github.com/jruby/jruby/issues/7182

**Comment by sashaioffe** [comment]
Looks like a known issue: https://bugs.openjdk.java.net/browse/JDK-8285445

Try to use command line option
-Djdk.io.File.enableADS=true

**Comment by robbavey** [comment]
Update to issue: 

This issue is not restricted to Zulu versions of the JDK, but is affected on any OpenJDK-derived version of the `8u331`, `11.0.15+10, `17.0.3+7` versions of the JDK on Windows.

**Comment by robbavey** [comment]
Fixed by #14066/#14085
